src/ape_networks/_cli.py

```diff
@@ -37,11 +37,13 @@ def make_sub_tree(data: Dict, create_tree: Callable) -> Tree:
             ecosystem_tree = make_sub_tree(ecosystem, Tree)
             _networks = ecosystem["networks"]
             for network in _networks:
-                network_tree = make_sub_tree(network, ecosystem_tree.add)
                 providers = network["providers"]
-                for provider in providers:
-                    make_sub_tree(provider, network_tree.add)
+                if providers:
```

Comment:
This makes it so `mainnet-fork` does `mainnet-fork` (or any provider, really) not show up if there are no providers in it.